### PR TITLE
Milestone2: Add Work Section

### PR DESCRIPTION
**Pull request by developerwaleed**
Milestone2: Add Work Section 

> Added image place holders.
> Added About me Section.
> Added Skills Section.
> Social icons are now clickable.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,41 +9,343 @@
...
+                                alt="Linkedin"></a></li>
+                    <li class="social-media"><a href="https://medium.com/"><img src="./icons/Social_icon3.png"
+                                alt="Medium"></a></li>
+                    <li class="social-media"><a href="https://angel.co/"><img src="./icons/Social_icon5.png"
+                                alt="angellist"></a></li>
+                </ul>
             </section>
         </div>
     </div>
+    <div class="container2">
```
Comment (youmari) [inline index.html]: this is a works section so it should be inside a `<section>`  tag , please try to do that

File: index.html [hunk 2]
```diff
@@ -8,41 +9,343 @@
...
+        <a href="#main-heading"><img class="arrow" src="./icons/down-arrow.svg" alt="Arrow"></a>
+        <h2 id="main-heading">My Recent <br>WORKS</h2>
+        <img id="indicator" src="./icons/Indicator.svg" alt="indicator">
+    </div>
+
+    <div class="card-holder">
```
Comment (youmari) [inline index.html:61]: I see that you implement grid CSS in the container2 but you should implement `display: grid;` in this section which is the one that holds the projects

File: index.html [hunk 3]
```diff
@@ -8,41 +9,343 @@
...
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
```
Comment (youmari) [inline index.html]: you should align the content of each project with  `flexbox` so please try to implement that

File: index.html [hunk 4]
```diff
@@ -8,41 +9,343 @@
...
+        <a href="#main-heading"><img class="arrow" src="./icons/down-arrow.svg" alt="Arrow"></a>
+        <h2 id="main-heading">My Recent <br>WORKS</h2>
+        <img id="indicator" src="./icons/Indicator.svg" alt="indicator">
+    </div>
+
+    <div class="card-holder">
+
+        <div class="card-holder-item">
+
+            <div class="holder-image">
```
Comment (youmari) [inline index.html:65]: this also should be an `` tag, I suggest to do that

File: index.html (hunk 3 above)
Comment (youmari) [inline index.html]: Your Design of the project card it's a bit large than the one in the Figma design, I suggest removing the white space and it should be the same as the Figma design 

- [ ] Figma design
image

- [ ] Your design 
image

File: index.html [hunk 5]
```diff
@@ -8,41 +9,343 @@
...
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
```
Comment (youmari) [inline index.html:89]: I see your button has the shadow on hover, not like the Figma design please try to stick to the Figma design

File: index.html [hunk 6]
```diff
@@ -8,41 +9,343 @@
...
+        <div id="title">
+            <h2>About me</h2>
+        </div>
+        <div class="about-me-body">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </div>
+        <div id="btn-2">
+            <button type="button" class="btn-2">Get My Resume</button>
```
Comment (youmari) [inline index.html:257]: I see that you forgot to implement the button style on hover and active, I suggest implementing that 
image

File: index.html [hunk 7]
```diff
@@ -8,41 +9,343 @@
...
+        </div>
+
+    </div>
+
+    <div class="about-me">
+
+        <div id="title">
+            <h2>About me</h2>
+        </div>
+        <div class="about-me-body">
```
Comment (youmari) [inline index.html]: Also, this should be a `<p>` tag, not a `<div>` tag that would be better if you do that

File: index.html [hunk 8]
```diff
@@ -8,41 +9,343 @@
...
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
+
+    </div>
+
+    <div class="about-me">
```
Comment (youmari) [inline index.html]: this is an about me section so it should be inside a `<section>` tag, please try to do that

File: index.html [hunk 9]
```diff
@@ -8,41 +9,343 @@
...
+                    <li class="skillstag-item">
+                        HTML
+                    </li>
+                    <li class="skillstag-item">
+                        CSS
+                    </li>
+                </ul>
+            </div>
+
+        </div>
```
Comment (youmari) [inline index.html:287]: Your skills cards having a big space at the bottom, please try to stick to the Figma design 

- [ ] Your design 
image

- [ ] Figma design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @caasperr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error
✔️ Nice Readme file 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (developerwaleed) [reply index.html:65]: Hey, can you please describe a little bit more what you want me to do? if you want me to add tag inside the <div> or you want me to replace the <div> with  tag? (if i replace  tag with <div> it leaves border which looks untidy so thats why its not possible to use img and also i cannot leave it without putting image and if i put any image in img tag that wont match the template)

Comment (developerwaleed) [reply index.html:65]: @youmari

**Review by zhadier — APPROVED** [review]
# Hi @caasperr 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great job implementing the features required by the previous reviewer



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (youmari) [reply index.html:65]: @caasperr I suggested you change it to an `` tag  cause it should be an image of the project

**Pull request merged**